### DISCUSSION TITLE
PIXI.Application.stage does not add duplicate sprites

### DISCUSSION THREAD

**Discussion by BroofTerr** [pixijs/pixijs#7886] [Q&A] [answered]
Hey!
I've started learning PixiJS and was doing the "BUILDING A PARALLAX SCROLLER WITH PIXI.JS" tutorial series when I ran into this problem: 

I made an object pool class and was taking some sprites to display them on the stage (had 2 types of tiles), but I noticed that only one sprite per type was being added to the stage.
Perhaps anyone has any suggestions on how to fix this, or am I doing something wrong (using the latest version of PixiJS).

Thank you for your time. Here are some screenshots of the code and browser console:
![image](https://user-images.githubusercontent.com/29431092/137642447-fd0f87a7-d061-437c-b8c8-65cc78dff888.png)
![image](https://user-images.githubusercontent.com/29431092/137642474-e1d959fd-c2d4-4864-a5ef-d832be6a88de.png)

**Comment by domis86** [comment, marked as answer]
Can you show also `takeWindow()` method?

**Reply by BroofTerr** [reply]
Hi, this is the entire pooling class.
But thankfully, found the problem :D needed to move the line 20 inside the loop, and now everything works (thought I could get away by creating that object once and just reuse'ing it inside the loop, but turns out it doesnt work like that)
![image](https://user-images.githubusercontent.com/29431092/137671170-a71be07b-d3c1-43c6-88f8-93a37184e0bf.png)
